src/scss/components/dropdown-select/mixins/SelectFacade.scss

```diff
@@ -2,40 +2,36 @@
 
 @mixin __selectFacade($focused: false) {
   @if $focused {
-    border: 1px solid $cyan-700;
+    border: 1px solid var(--hxDropdown-focused-borderColor, $cyan-700);
     box-shadow: $focus-glow;
-  }
-  @else {
-    border: 1px solid $gray-500;
-    color: $gray-900;
+  } @else {
+    border: 1px solid var(--hxDropdown-unfocused-borderColor, $gray-500);
+    color: var(--hxDropdown-color, $gray-900);
   }
 }
 
 @mixin __selectFacade--invalid($focused: false) {
   @if $focused {
-    border: 2px solid $red-900;
+    border: 2px solid var(--hxDropdown-invalid-focused-borderColor, $red-900);
```

Comment:
Please update to the new Color Palette 2.0 color variables.

Reply:
Changes have been done, please review.